### PR TITLE
bugfix:  can remove the exited container without flag force.

### LINKED ISSUE #639: [bug] should also check exited state when remove container
**Issue Description**
In `daemon/mgr/container.go`, function Remove, line 166, it should also check exited state when remove container:
```
// Remove removes a container, it may be running or stopped and so on.
func (mgr *ContainerManager) Remove(ctx context.Context, name string, option *ContainerRemoveOption) error {
	c, err := mgr.container(name)
	if err != nil {
		return err
	}
	c.Lock()
	defer c.Unlock()

	if !c.IsStopped() && !c.IsCreated() && !option.Force {
		return fmt.Errorf("container: %s is not stopped, can't remove it without flag force", c.ID())
	}

```
**Describe what happened**:
It failed when remove `exited` container:
```
#pouch ps -a
Name                      ID       Status   Created          Image                                            Runtime
test-run-device-mapping   e8d87b   exited   14 seconds ago   registry.hub.docker.com/library/busybox:latest   runc

#pouch rm test-run-device-mapping
Error: failed to remove container: {"message":"container: e8d87b96ff429d00aa7072a6ecdab4946d94cddd728714ffa50ba56b4c71e5af is not stopped, can't remove it without flag force"}
```

**Describe what you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:
1.
2.
3.

**Anything else we need to know?**:

**Environment**:
- pouch version (use `pouch version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by skoo87**
Signed-off-by: skoo87 <marckywu@gmail.com>

**1.Describe what this PR did**

**2.Does this pull request fix one issue?** 

fix #639 

**3.Describe how you did it**

**4.Describe how to verify it**

**5.Special notes for reviews**

### CODE REVIEW

**Comment by Letty5411** [comment]
LGTM

File: daemon/mgr/container.go [hunk 1]
```diff
@@ -163,7 +163,7 @@ func (mgr *ContainerManager) Remove(ctx context.Context, name string, option *Co
 	c.Lock()
 	defer c.Unlock()
 
-	if !c.IsStopped() && !c.IsCreated() && !option.Force {
+	if !c.IsStopped() && !c.IsExited() && !c.IsCreated() && !option.Force {
 		return fmt.Errorf("container: %s is not stopped, can't remove it without flag force", c.ID())
```
Comment (Letty5411) [inline daemon/mgr/container.go]: Do we need to also change the error message?

Comment (skoo87) [reply daemon/mgr/container.go]: I think it's unnecessary ....

**Comment by allencloud** [comment]
LGTM

File: daemon/mgr/container.go (hunk 1 above)
Comment (allencloud) [reply daemon/mgr/container.go]: It is all right.

**Pull request merged**